Dockerfile.openpilot_base

```diff
@@ -24,7 +24,7 @@ RUN cd /tmp && \
 
 ENV POETRY_VIRTUALENVS_CREATE=false
 ENV PYENV_VERSION=3.11.4
-ENV PYENV_ROOT="/root/.pyenv"
+ENV PYENV_ROOT="/opt/pyenv"
```

Comment:
if we move the pyenv to /opt, it will still allow for the CI to work normally, but the devcontainer can chown it as batman and allow for writes

Reply:
There is no problem with user id 1000 always owning the files.  Create the user with id 1000 with `RUN useradd -m -s /bin/bash -u 1000 <username>` (use batman if you like, all that matters is the user id) and set it active with `USER <username>`, then when you install it will already be owned by that user (I would put `PYENV_ROOT` in `/home/<username>/.pyenv` to keep things simple).  This should allow you to delete what you added to `.devcontainer/container_post_create.sh`.  If we still want the container to run as root by default you can issue another USER command at the end to switch back to root.

Reply:
done